go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/0chain/errors v1.0.3
-	github.com/0chain/gosdk v1.5.1-0.20220130180707-ab6ebf190c99
+	github.com/0chain/gosdk v1.5.1-0.20220210040637-95f825c1c3ae
```

Comment:
Can we make a release of this gosdk version?

Reply:
it is not worth to do it now. because all related PRs are merged at the same time. so the release is published from the same commit id too.